SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -117,6 +117,11 @@ + (UIColor *)ows_messageBubbleLightGrayColor
     return [UIColor colorWithHue:240.0f / 360.0f saturation:0.02f brightness:0.92f alpha:1.0f];
 }
 
++ (UIColor *)ows_tableCellSelectedBackgroundColor
```

Comment:
I don't think we should have every single color constant defined here, especially for constants that we only use in a single place.  It'll make this category enormous, and I don't see any benefit.

Reply:
ok, I'll take it out.